docs/src/pages/style/color.md

```diff
@@ -25,20 +24,46 @@ They’ve been designed to work harmoniously with each other.
 For instance, you can refer to complementary primary and accent colors (for example 'red 500' & 'purple A200'), like so:
 
 ```js
-import { red, purple } from 'material-ui/colors';
+import purple from 'material-ui/colors/purple';
+import red from 'material-ui/colors/red';
 
 const primary = red[500]; // #F44336
 const accent = purple['A200']; // #E040FB
 const accent2 = purple.A200; // #E040FB (alternative method)
 ```
 
-### Full Palette
-{{"demo": "pages/style/Color.js"}}
+## Color tool
+
+The material design team has built an awesome palette configuration tool: [https://material.io/color](material.io/color).
+It's gonna help you create, share, and apply color palettes to your UI, as well as measure the accessibility level of any color combination.
```

Comment:
"It will help you create color palettes for your UI," ("gonna" is slang, so fine for chat, but not for the docs). Also not convinced about the "sharing" and "applying" part.